tests/Dummy/CreateChannelAction.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace BrandEmbassyTest\Slim\Dummy;
+
+use BrandEmbassy\Slim\ActionHandler;
+use BrandEmbassy\Slim\Request\RequestInterface;
+use BrandEmbassy\Slim\Response\ResponseInterface;
+
+class CreateChannelAction implements ActionHandler
```

Comment:
toto je tu mozna navic?

Reply:
Je to dummy testovací kod použitý v testu